addons/project/models/project.py

```diff
@@ -289,8 +289,7 @@ def _get_default_favorite_user_ids(self):
     def _read_group_stage_ids(self, stages, domain, order):
         return self.env['project.project.stage'].search([], order=order)
 
-    name = fields.Char("Name", index='trigram', required=True, tracking=True, translate=True, default_export_compatible=True,
-        help="Name of your project. It can be anything you want e.g. the name of a customer or a service.")
+    name = fields.Char("Name", index='trigram', required=True, tracking=True, translate=True, default_export_compatible=True)
```

Comment:
It is not a bug we could remove it in master.

Reply:
hello @xavierbol ,
its already removed in master
ref taskID-3073860
Thanks :slightly_smiling_face: